meta-genivi-dev/recipes-cdl/cdl-concept-demo/cdl-concept-demo.bb

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2015-2016 GENIVI Alliance
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+LICENSE  = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=9741c346eef56131163e13b9db1241b3"
+SRC_URI = "git://github.com/GENIVI/car-data-logger.git;branch=proof-of-concept;protocol=https"
+SRCREV  = "14a44c1d0db2b02582bcdf9e38b0947677c6f485"
+
+SUMMARY = "CDL Concept Demo"
+DEPENDS = "qtbase qtdeclarative vsomeip common-api-c++-someip dbus dlt-daemon common-api-c++-dbus common-api-c++"
+
+S = "${WORKDIR}/git"
+
+inherit qmake5
+
+FILES_${PN} += "/opt/cdl/*"
+INSANE_SKIP_${PN} = "dev-so"
```

Comment:
There is an error during do_package_qa. Error is "non dev package contains symlink."
Should I install files manually?